src/lib.rs

```diff
@@ -27,8 +25,19 @@ mod serializer;
 mod tree;
 
 pub use attributes::Attributes;
-pub use iter::{NodeIterator, ElementIterator};
 pub use node_data_ref::NodeDataRef;
-pub use parser::{Html, ParseOpts};
+pub use parser::{parse_html, ParseOpts};
 pub use select::Selectors;
 pub use tree::{NodeRef, Node, NodeData, ElementData, Doctype, DocumentData};
+
+/// This module re-exports a number of traits that are useful when using Kuchiki.
+/// It can be used with:
+///
+/// ```rust
+/// use kuchiki::traits::*;
+/// ```
+pub mod traits {
+    pub use iter::{NodeIterator, ElementIterator};
+    pub use html5ever::tendril::TendrilSink;
```

Comment:
I’ve made html5ever re-export the tendril crates, so that users like kuchiki don’t need to depend on it themselves. That way, when html5ever upgrades to a semver-incompatible version of tendril, kuchiki will automatically use the same version. Previously, the version numbers of both dependencies had to be kept in sync.